b2sdk/v1/file_version.py

```diff
@@ -119,9 +126,29 @@ def inner(*a, **kw):
 class FileVersionInfoFactory(v2.FileVersionFactory):
 
     from_api_response = translate_single_file_version(v2.FileVersionFactory.from_api_response)
-    from_response_headers = translate_single_file_version(
-        v2.FileVersionFactory.from_response_headers
-    )
+
+    def from_response_headers(self, headers):
+        file_info = {}
+        prefix_len = len(FILE_INFO_HEADER_PREFIX_LOWER)
+        for header_name, header_value in headers.items():
+            if header_name[:prefix_len].lower() == FILE_INFO_HEADER_PREFIX_LOWER:
+                file_info_key = header_name[prefix_len:]
+                file_info[file_info_key] = header_value
```

Comment:
duplicate of `DownloadVersionFactory.from_response_headers()`. Maybe move out extraction of `file_info` dict from headers to a separate function?